### ISSUE TITLE
cannot find html theme file

### ISSUE THREAD

**Issue by rumpelsepp** [gauteh/astroid#70]
I managed to run astroid on arch linux (I'll have to send a few fixes for the PKGBUILD, though). When I want to view a thread I get this:

```
[ERROR] (15:15:05): tv: cannot find html theme file: ui/thread-view.html, using default..
[ERROR] (15:15:05): tv: cannot find default html theme file.
```

**Comment by gauteh** [comment]
Stefan Tatschner writes on January 19, 2016 15:16:
> I managed to run astroid on arch linux (I'll have to send a few fixes for the PKGBUILD, though). When I want to view a thread I get this:
> 
> ```
> [ERROR] (15:15:05): tv: cannot find html theme file: ui/thread-view.html, using default..
> [ERROR] (15:15:05): tv: cannot find default html theme file.
> ```

I believe this was just fixed in #69, please try to rebuild the
PKGBUILD.

**Comment by rumpelsepp** [comment]
Yeah, it's fixed. Thx!
